.github/workflows/continuous-integration-workflow.yml

```diff
@@ -82,6 +93,7 @@ jobs:
                 ct-ng olddefconfig; \
                 ct-ng source; \
           done
+          find ./.local/src -type f -not -newer stamp -delete -print
```

Comment:
Which means this won't do anything because everything will be newer than `stamp`